network/src/peers/peer/inbound_handler.rs

```diff
@@ -60,6 +60,10 @@ impl Peer {
                 metrics::increment_counter!(PINGS);
                 metrics::increment_counter!(ALL_SUCCESSES);
```

Comment:
Suggestion: modify the import (or comment) so that it's obvious this is inbound::ALL_SUCCESSES.